README.md

```diff
@@ -22,6 +22,7 @@ runners are supported:
 |       Language | Test Runners                                                                                                       | Identifiers                                                                                                                                  |
 | -------------: | :----------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------- |
 |         **C#** | .NET                                                                                                               | `xunit`, `dotnettest`                                                                                                                        |
+|         **C++** | CTest,Make                                                                                                               | `ctest`, `make`                                                                                                                        |
```

Comment:
This just adds support for Catch2 right? If so you want to enter 

```md
**C++** | Catch2  | `catch2`
```

Reply:
Hmm, so the issue is that in C++, the testing framework doesn't come with a test runner, and instead you have to build it. One implication of this is that the test runner's name is whatever the target's name is in the build system of choice(which I defaulted to `make`), so you can't call `catch2` on the command line like you can with say `pytest`, or `cargo test`.

Let me know what you think the best way is to make this info clear to future readers!

Reply:
[Every runner should add a new Vim command for invoking it](https://github.com/vim-test/vim-test#runner-commands). If you first run `let g:test#runner_commands = ['Catch2']` are you able to invoke the command?

Reply:
Hi @codeinabox , looks like I was overthinking it, as adding that does indeed allow me to invoke the test runner from within Vim, which makes sense as from reading the code it calls the same `test#executable` function as the general vim-test commands do. I added the command runner's usage in the recent commit below. Can you review?